features/vesting/vestingCardDetailed.tsx

```diff
@@ -0,0 +1,121 @@
+import { format, isPast } from 'date-fns';
+import {
+  useVestingCliff,
+  useVestingEndTime,
+  useVestingLocked,
+  useVestingToken,
+  useVestingUnclaimed,
+  Vesting,
+} from 'features/vesting';
+import { FC, memo } from 'react';
+import { CarouselCard, FormatToken } from 'shared/ui';
+import {
+  Header,
+  Badge,
+  Index,
+  Address,
+  Details,
+  Row,
+  Column,
+  DetailsHeader,
+  DetailsValue,
+  CustomLoader,
+} from './vestingCardStyles';
+
+export type VestingCardDetailsProps = {
+  index?: number;
+  vesting?: Vesting;
+};
+
+const formatDate = (date: Date | undefined): string => {
+  if (date == null) {
```

Comment:
please either use === undefined or extend argument types scope

Reply:
it's generic check for null or undefined, since nullish value can be passed in js and bypass ts validations